src/components/sidecar/get-truncated-title.ts

```diff
@@ -0,0 +1,27 @@
+const MAX_CHARACTERS = 26
+
+function getTruncatedTitle(fullTitle: string): string {
+  let truncatedTitle: string
+
+  if (fullTitle.length < MAX_CHARACTERS) {
+    truncatedTitle = fullTitle
+  } else {
+    let characterCount = 0
+    const words = fullTitle.split(' ')
+    const wordsToInclude = []
+    words.forEach((word, idx) => {
+      const wordLength = word.length
+      if (characterCount + wordLength < MAX_CHARACTERS) {
+        wordsToInclude.push(word)
+        characterCount += wordLength
+        // count spaces as well, since these are counted
+        // in the fullTitle.length comparison
+        if (idx !== words.length) characterCount++
```

Comment:
Fixes a nitpicky issue I noticed on the [current UI assembly preview](https://dev-portal-git-assembly-ui-v1-hashicorp.vercel.app/waypoint/docs). The title `contrib Tools and Helpers` ends up being "truncated" into the `contrib Tools and Helpers...`, I think counting spaces consistently fixes this 👍.

Very related: I bumped up the `MAX_CHARACTERS` from 25 to 26 to counteract the above change, as `contrib Tools and Helpers` felt like it should pass our "short enough to not truncate" bar.